test.yaml

```
@@ -0,0 +1,8 @@
+spire-server:
+  upstreamAuthority:
+    certManager:
+      enabled: true
+      # -- Creates a Cert-manager CA
+      createCA: true
```

```suggestion
      ca:
        create: true
```

According to https://github.com/spiffe/helm-charts/pull/342/files#diff-2970cff31c683ad68e57af57ddbb9acf98765a7898efdeff84c8532c260e1dcbR196-R198